### PR TITLE
Allow selection of group classifier

### PR DESCRIPTION
**Pull request by WilliamBZA**
Adds support for displaying groups displayed by different classifications.

Addresses: https://github.com/Particular/ServicePulse/issues/239
Addresses: https://github.com/Particular/PlatformDevelopment/issues/1038

Ping @Particular/servicepulse-maintainers, @janovesk, @mikeminutillo, @SzymonPobiega, @tmasternak, @boblangley

### CODE REVIEW

**Comment by WojcikMike** [comment]
LGTM

**Comment by WilliamBZA** [comment]
NB: This should only be merged and released once SC 1.27.0 has been released.

File: src/ServicePulse.Host/app/js/views/failed_groups/view.html [hunk 1]
```diff
@@ -15,9 +15,24 @@
 
         <no-data ng-show="vm.exceptionGroups.length === 0 && !vm.loadingData" title="message groups" message="There are currently no grouped message failures"></no-data>
 
-        <div class="row">
-            <div class="col-sm-12" ng-show="vm.exceptionGroups.length > 0">
+        <div class="row" ng-show="vm.availableClassifiers.length && vm.exceptionGroups.length">
```
Comment (tmasternak) [inline src/ServicePulse.Host/app/js/views/failed_groups/view.html]: This condition means that if we switch to classification that has no groups we won't be able to go back, right?

I think we should remove `vm.exceptionGroups.length` part.

Comment (WilliamBZA) [inline src/ServicePulse.Host/app/js/views/failed_groups/view.html]: Good catch. I've updated.

**Comment by WilliamBZA** [comment]
Changed from [WIP] to ready as SC has been released.

@tmasternak We just need the release notes now and we can release this.

**Comment by tmasternak** [comment]
@WilliamBZA I think that considering final version we have chosen for SC (reusing the already existing classifications) I don't think it's critical to release SP ASAP. In other words this could be bundled with next SP release when maintainers decide to do a release.

/cc: @mikeminutillo

**Pull request merged**